posts/2015-04-20-mix-it-2015-pt1.md

```diff
@@ -0,0 +1,50 @@
+
+[Plus de détails ici](https://docs.google.com/document/d/1bYMyE6NdiAupuwl7pWQfB-vOZBPSsXCv57hljLDMV8E)
+
+
+## Le succès Lichess.org
+
+[Lichess.org](http://fr.lichess.org/) est un site de jeu d'échecs en ligne.
+
+Le projet est open-source, gratuit à 100% pour tous ses utilisateurs et sans publicité !
+En quelques années, il a réussi à se hisser en deuxième position du classement mondial des site d'échecs en ligne, face à de nombreux sites à but lucratifs portés par de grosses sociétés.
+
+*Comment arrive-t-il à surpasser ses concurrents sans avoir leurs moyens ?*
+
+- Domination technique : proposer un service plus rapide, plus performant, où l'utilisateur attend moins (l'importance de la perfomance... ;)).
+- Qualité du service proposé : donner aux utilisateurs des fonctionalités pertinentes, qu'ils attendent (ici plus de 10 variantes du jeu d'échecs, l'analyse de ses parties, le jeu contre l'ordinateur, ...). Le site bénéficie des conseils de la part de joueurs d'échecs de haut niveau et tient compte de leur avis d'expert.
+- Rapidité / Agilité : itérer rapidement pour proposer souvent du contenu à jour, s'adapter rapidement, expérimenter en réduisant le coût de l'échec.
+
+##Javascript
+
+Javascript a été à l'honneur durant cette édition du Mix-It puisque pas moins de cinq sessions étaient consacrées au langage ou à des outils issus de ce langage :
+
+- deux ateliers qui abordaient respectivement le développement d'applications mobiles avec [Appcelerator Titanium](http://www.mix-it.fr/session/2321/build-native-mobile-apps-using-js-and-appcelerator) et la découverte des nouveautés Ecmascript ([Le JavaScript du futur au bout des doigts](http://www.mix-it.fr/session/1651/le-javascript-du-futur-au-bout-des-doigts)) ;
+
+- une conférence intitulée [Changez d'avis sur Javascript](http://www.mix-it.fr/session/1351/changez-d-avis-sur-javascript) durant laquelle Damien Feugas a présenté quelques-unes des nouveautés introduites par les spécifications d'EcmaScript 6 'Harmony', parmi lesquelles la déclaration de classes Objet calquée sur la plupart des langages objets (Java, C#, etc.), l'introduction du mot-clé 'super', la fonction 'Flèche' qui permet de conserver le contexte 'this' à l'intérieur des fonctions, et quelques autres fonctionnalités utiles. Le conférencier n'a malheureusement pas pu aborder des concepts importants tels que les Promesses et les Générateurs, sans doute faute de temps, mais il n'en demeure pas moins que la conférence était très didactique et très agréable à suivre. Noter que la publication officielle de cette spécification devrait intervenir au cours de l'année 2015, et que, pour l'heure, la plupart des navigateurs et librairies serveur (Node.js, Io.js, etc.) ne l'implémentent que partiellement ;
```

Comment:
Too many commas kill the comma spirit ;-) `année 2015, et que, pour l'heure, la plupart`.